### PR TITLE
Add scripts for running gvisor individual test suites for ngo and occlum

### PR DESCRIPTION
**Pull request by daan3601**

### CODE REVIEW

File: occlum/run_test_case.sh [hunk 1]
```diff
@@ -0,0 +1,112 @@
+
```
Comment (jessehui) [inline occlum/run_test_case.sh]: Remove the first empty line.

File: occlum/run_test_case.sh [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+#!/bin/bash
+TEST_CASE_DIR=/opt/occlum/gvisor_syscall_tests
+BLOCKLIST_DIR=../blocklist
+NGO_BLOCKLIST_DIR=../ngo_blocklist
+FILTER=""
+NGO_FILTER=""
+VERSION=""
+num1=0
```
Comment (jessehui) [inline occlum/run_test_case.sh]: Please rename the variable with a more precise name.

File: occlum/run_test_case.sh [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+do
+	case "$1" in
+		-s)
+			if [ -z $VERSION ];then
+				echo "Missing arguments, you can use '-w occlum' to run test cases that pass in occlum or '-w ngo' to run test cases that pass in ngo"
+				exit -1
+			fi
+			shift 1
+			num1=`expr $num1 + 1`
+			if [ `expr $numl % 2` -eq 0 ];then
```
Comment (jessehui) [inline occlum/run_test_case.sh]: what is `$numl` ? Please check the spelling. And find out why this error is not met by you.

File: occlum/run_test_case.sh [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+				run_testcase $testcase
+			fi
+			testcase=$1
+			shift 1;;
+
+		-t)
```
Comment (jessehui) [inline occlum/run_test_case.sh:71]: what will happen if the user only input `./run_test_case.sh -t UdpBindTest/SendtoTest.Sendto/0`?

**Review by jessehui — COMMENTED** [review]
In general, this looks good. Only some minor mistakes.

File: occlum/run_test_case.sh (hunk 1 above)
Comment (daan3601) [reply occlum/run_test_case.sh]: Modification complete

File: occlum/run_test_case.sh (hunk 2 above)
Comment (daan3601) [reply occlum/run_test_case.sh]: Modification complete

File: occlum/run_test_case.sh (hunk 4 above)
Comment (daan3601) [reply occlum/run_test_case.sh:71]: Modification complete

File: occlum/run_test_case.sh (hunk 3 above)
Comment (daan3601) [reply occlum/run_test_case.sh]: Modification complete

**Review by jessehui — APPROVED** [review]
LGTM

**Pull request closed without merging**